rerun_js/web-viewer/build-wasm.mjs

```diff
@@ -31,7 +31,8 @@ function buildWebViewer(mode) {
       "cargo run -p re_dev_tools -- build-web-viewer",
       modeFlags,
       "--target no-modules-base",
-      "--features grpc",
+      "--no-default-features",
+      "--features grpc,map_view",
```

Comment:
I suggest we have a `release = ["grpc", "map_view"]` feature in `re_viewer` instead. It feels like a more discoverable location for controlling what goes into the release.

```suggestion
      "--features release",
```

Reply:
At this point I just made `map_view` the default. I don't agree with the `release` feature for web viewer specifically, it makes sense for rerun-cli because there it makes it possible for it to be compiled without `nasm`, and with `-F release` in case you do want `nasm`.

Reply:
The big problem here imho was `build_web_viewer` silently passing `--no-default-features` to the `re_viewer` build. To discover all the features being requested, you look through the commands that mention `build-web-viewer`, same as with every other non-web build command.

Reply:
Makes sense 👍